### ISSUE TITLE
Translated collection instances are exported without translation

### ISSUE THREAD

**Issue by tdaniel22** [KhronosGroup/glTF-Blender-IO#665]
**Describe the bug**
Translated collection instances are exported without translation.

**To Reproduce**
1. Create a cube and put it in a new collection.
2. Create an instance of this new collection.
3. Translate, rotate and scale the new instance.
4. Export the scene.

**Expected behavior**
The exported glTF node corresponding to the cube instance should be translated, rotated and scaled.
```
"nodes" : [
    {
        "mesh" : 0,
        "name" : "Cube"
    },
    {
        "children" : [
            0
        ],
        "name" : "Duplication_Offset_CubeInstance",
        "rotation" : [
            0.3826834559440613,
            0,
            0,
            0.9238795042037964
        ],
        "scale" : [
            1.5,
            1.5,
            1.5
        ],
        "translation" : [
            3,
            0,
            0
        ]
    }
]
```

**Observed behavior**
The exported instance node does not contain a translation.
```
"nodes" : [
    {
        "mesh" : 0,
        "name" : "Cube"
    },
    {
        "children" : [
            0
        ],
        "name" : "Duplication_Offset_CubeInstance",
        "rotation" : [
            0.3826834559440613,
            0,
            0,
            0.9238795042037964
        ],
        "scale" : [
            1.5,
            1.5,
            1.5
        ]
    }
]
```

**Version**
 - OS: Linux
 - Blender Version 2.80

**Additional context**
In [`gltf2_blender_gather_nodes.py`](https://github.com/KhronosGroup/glTF-Blender-IO/blob/368c80402a59e2df0b6ec42ef6584c10c86f59ba/addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_nodes.py#L348), instead of using the "normal" translation, collections are translated by the [`instance_offset`](https://docs.blender.org/api/current/bpy.types.Collection.html#bpy.types.Collection.instance_offset) vector. Commenting out those lines fixes the problem. I'm not sure exactly what the `instance_offset` property corresponds to in Blender. Maybe both vectors should be combined?

**Comment by julienduroure** [comment]
Hello,
Thanks for reporting.
This is linked to #500 . Currently, duplication / instance is quite buggy

**Comment by julienduroure** [comment]
@tdaniel22  This is solved by PR #950.
Target is to merge it soon (and will be available in blender 2.83)
